README.md

```diff
@@ -21,22 +20,193 @@ One line of code per option is all you need to write. For that, you get a nice
 automatically-generated help page, robust option parsing, and sensible defaults
 for everything you don't specify.
 
+This code is a feature-fork of Optimist: https://github.com/ManageIQ/optimist
```

Comment:
TODO: Need to blend this into the main README, instead of being a separate section.